src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -381,9 +381,7 @@ public function present($cart, $shouldSeparateGifts = false)
             $taxAmount = $total_including_tax - $total_excluding_tax;
             $subtotals['tax'] = array(
                 'type' => 'tax',
-                'label' => ($this->includeTaxes())
-                    ? $this->translator->trans('Included taxes', array(), 'Shop.Theme.Checkout')
-                    : $this->translator->trans('Taxes', array(), 'Shop.Theme.Checkout'),
+                'label' => $this->translator->trans('Included taxes', array(), 'Shop.Theme.Checkout'),
```

Comment:
This will only be correct if the option to display prices _with taxes_ is activated.

Reply:
Should I put back the former function?

Reply:
Yes the label adapts depending on `$this->includeTaxes()`

Reply:
Okay

Reply:
No, I think that you have to show it only when the price _includes_ tax. For example, prices in the US never include tax, but it doesn't mean there are no taxes -- they are just shown separately.

Reply:
In the end the user pays with the taxes right ? So usually we show the price tax excluded and then the price with taxes included. This is why we have a line below saying "Included taxes"